code_schemes/somalia_operator.json

```diff
@@ -0,0 +1,91 @@
+{
+  "SchemeID": "Scheme-f86cff0b",
+  "Name": "Somalia Operator",
+  "Version": "0.0.0.2",
+  "Codes": [
+    {
+      "CodeID": "code-450ec146",
+      "CodeType": "Normal",
+      "DisplayText": "golis",
+      "NumericValue": 1,
+      "StringValue": "golis",
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "golis"
+      ]
+    },
+    {
+      "CodeID": "code-c7b32481",
+      "CodeType": "Normal",
+      "DisplayText": "hormud",
```

Comment:
Can I check the spelling here - I thought it was hormuud

Reply:
Past experience at AVF is that Somali isn't always very well standardised in latin script. I will check with the project team.

Reply:
Officially it's hormuud, but not going to change here - we use this spelling throughout D&T and on all previous projects in Somalia